### PR TITLE
adding validations to avoid duplicate productCategory, and adding fri…

### PR DESCRIPTION
**Pull request by rocha7778**
…endly message error

### CODE REVIEW

File: service/mantle/shopify/ApiServices.xml [hunk 1]
```diff
@@ -325,26 +325,40 @@
...
             </entity-find-one>
+
+            <if condition="!productCategory">
+                <set field="error" value="true"/>
+                <message type="warning">The product category ${productCategoryId} does not exist</message>
+                <return/>
+           </if>
+           <if condition="productCategory.pseudoId != productCategory.productCategoryId">
+                <set field="error" value="true"/>
+                <message type="warning">The product category ${productCategoryId} already exist</message>
```
Comment (marror7) [inline service/mantle/shopify/ApiServices.xml:342]: already exists*

Comment (marror7) [inline service/mantle/shopify/ApiServices.xml:342]: already exists, please correct

File: service/mantle/shopify/ApiServices.xml [hunk 2]
```diff
@@ -325,26 +325,40 @@
...
+           </if>
+
             <entity-find entity-name="mantle.product.category.ProductCategoryMember" list="productCategoryMembers">
                 <econdition field-name="productCategoryId"/>
                 <econdition field-name="thruDate" from="null"/>
             </entity-find>
             <script>import groovy.json.JsonSlurper</script>
             <set field="shopifyCollectItems" from="[]"/>
+            <!-- 
+                if the product category has members those member are added to current category
```
Comment (marror7) [inline service/mantle/shopify/ApiServices.xml:353]: Those members*

**Review by marror7 — CHANGES_REQUESTED** [review]

File: service/mantle/shopify/ApiServices.xml (hunk 1 above)
Comment (rocha7778) [reply service/mantle/shopify/ApiServices.xml]: Done

File: service/mantle/shopify/ApiServices.xml (hunk 2 above)
Comment (rocha7778) [reply service/mantle/shopify/ApiServices.xml]: Done

**Review by marror7 — APPROVED** [review]

**Pull request merged**